### PR TITLE
Fix testGetMicrosoftTimeZoneNameBad() test case

### PR DESCRIPTION
**Pull request by vboctor**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vboctor__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java [hunk 1]
```diff
@@ -52,10 +52,11 @@ public void testGetMicrosoftTimeZoneNameBad() {
     } catch (final IllegalArgumentException ignored) {}
```
Comment (serious6) [inline src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java]: while looking at this I was wondering what we want to do here? `Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));` I think this test should handle a `@Test(expected = IllegalArgumentException.class` for the null vlaue

**Comment by serious6** [comment]
@vboctor Maybe this test should be splitted and annotated with `@Test(expected = ...)` what do you think?

**Comment by serious6** [comment]
@vboctor is this PR still needed?

**Pull request closed without merging**